docs/samples/fluid/README.md

```diff
@@ -0,0 +1,343 @@
+
+It is important to note that this PoC has not been tested in a production environment and is intended for experimental purposes only. Therefore, if you decide to try it out, please carefully validate it in your use case before deploying it in a production environment.
+## Context
+
+With the increasing popularity of large language models (LLMs), many companies are now deploying them via KServe. However, due to their typical size ranging from a few GBs to several hundred GBs, it can take a considerable amount of time for the KServe Storage Initializer to pull them from remote storage. As a result, the start-up time for an inference service can take anywhere from a few minutes to over 30 minutes due to the lengthy model downloading and loading times. This makes the serverless autoscaling function of KServe less effective for LLM use cases.
+To improve performance, it becomes crucial to cache the model artifacts rather than pulling them from remote storage during the autoscaling process. Since large language models are often served on GPU instances, which can be expensive, autoscaling becomes a helpful tool for reducing costs by allowing resources to be scaled up or down based on demand.
+
+## Prerequisite
+
+* [KServe](https://kserve.github.io/website/master/admin/serverless/serverless/)
+  * Please follow this [guideline](https://kserve.github.io/website/master/developer/developer/#deploy-kserve-from-master-branch) to deploy KServe from `master` branch
+  * Enable node selector in KServe InferenceService via KNative
+
+    ```sh
+
+    kubectl patch configmap/config-features \
+      --namespace knative-serving \
+      --type merge \
+      --patch '{"data":{"kubernetes.podspec-nodeselector":"enabled"}}'
+    ```
+  * Enable Direct VolumeMount for PVC In KServe
+
+    ```sh
+    # edit inferenceservice-config and update enableDirectPvcVolumeMount to true
+    kubectl edit configmap inferenceservice-config  -n kserve
+
+    #   storageInitializer: |-
+    #     {
+    #         "image" : "kserve/storage-initializer:latest",
+    #         "memoryRequest": "100Mi",
+    #         "memoryLimit": "1Gi",
+    #         "cpuRequest": "100m",
+    #         "cpuLimit": "1",
+    #         "storageSpecSecretName": "storage-config",
+    #         "enableDirectPvcVolumeMount": false        # change to true
+    #     }
+    ```
+
+* [Fluid](https://github.com/fluid-cloudnative/fluid/blob/master/docs/en/userguide/get_started.md)
+  * Deploy Fluid from `master` branch
+
+    ```sh
+    # Deploy Fluid from master branch
+    git clone https://github.com/fluid-cloudnative/fluid
+
+    # if you are using alluxio runtime
+    helm upgrade --install \
+        --set runtime.alluxio.enabled=true \
+        --set webhook.reinvocationPolicy=IfNeeded \
+        fluid --create-namespace --namespace=fluid-system fluid/charts/fluid/fluid
+
+    # if you are using jindo runtime (this example)
+    helm upgrade --install \
+        --set runtime.jindo.enabled=true \
```

Comment:
By default, you don't have to set `runtime.jindo.enabled=true`, the runtime controller will be created according to the runtime type.